data/explorations/android_interactions/Test all android interactions.yaml

```diff
@@ -0,0 +1,801 @@
+        rule_specs:
+        - inputs:
+            x:
+            - - ca_choices_0
+              - ca_choices_1
+            - - ca_choices_3
+            - - ca_choices_2
+          rule_type: IsEqualToOrdering
+        tagged_skill_misconception_id: null
+        training_data: []
+      - outcome:
+          dest: drag and drop sort
+          feedback:
+            content_id: feedback_5
+            html: <p>Seems like you did the ascending order</p>
+          labelled_as_correct: false
+          missing_prerequisite_skill_id: null
+          param_changes: []
+          refresher_exploration_id: null
+        rule_specs:
+        - inputs:
+            x: ca_choices_0
+            y: 2
+          rule_type: HasElementXAtPositionY
+        tagged_skill_misconception_id: null
+        training_data: []
+      confirmed_unclassified_answers: []
+      customization_args:
+        allowMultipleItemsInSamePosition:
+          value: true
+        choices:
+          value:
+          - content_id: ca_choices_0
+            html: <p>3/5</p>
+          - content_id: ca_choices_1
+            html: <p>6.0</p>
+          - content_id: ca_choices_2
+            html: <p>0.35</p>
+          - content_id: ca_choices_3
+            html: <p>0.46</p>
+      default_outcome:
+        dest: drag and drop sort
+        feedback:
+          content_id: default_outcome
+          html: <p>Not quite. Try again.</p>
+        labelled_as_correct: false
+        missing_prerequisite_skill_id: null
+        param_changes: []
+        refresher_exploration_id: null
+      hints: []
+      id: DragAndDropSortInput
+      solution: null
+    linked_skill_id: null
+    next_content_id_index: 6
+    param_changes: []
+    recorded_voiceovers:
+      voiceovers_mapping:
+        ca_choices_0: {}
+        ca_choices_1: {}
+        ca_choices_2: {}
+        ca_choices_3: {}
+        content: {}
+        default_outcome: {}
+        feedback_4: {}
+        feedback_5: {}
+    solicit_answer_details: false
+    written_translations:
+      translations_mapping:
+        ca_choices_0: {}
+        ca_choices_1: {}
+        ca_choices_2: {}
+        ca_choices_3: {}
+        content: {}
+        default_outcome: {}
+        feedback_4: {}
+        feedback_5: {}
+  end:
+    card_is_checkpoint: false
+    classifier_model_id: null
+    content:
+      content_id: content
+      html: Congratulations, you have finished!
+    interaction:
+      answer_groups: []
+      confirmed_unclassified_answers: []
+      customization_args:
+        recommendedExplorationIds:
+          value: []
+      default_outcome: null
+      hints: []
+      id: EndExploration
+      solution: null
+    linked_skill_id: null
+    next_content_id_index: 0
+    param_changes: []
+    recorded_voiceovers:
+      voiceovers_mapping:
+        content: {}
+    solicit_answer_details: false
+    written_translations:
+      translations_mapping:
+        content: {}
+  image region:
```

Comment:
Can we use the already created solar system image interaction data? The assets are already there in repo and this way we can directly use the existing resources and we don't need to add new images. 

https://github.com/oppia/oppia/blob/14d9f1e8b59fd6dd5c3ccea8fe4fb903379e20f9/data/explorations/solar_system/The%20Solar%20System.yaml#L295

Reply:
Are you using the link in the description to load the exploration?
Actually loading the exploration from the community library uses the GCS, but we need the AssetDevHandler here.
This is the same for all other dummy explorations.

Reply:
Assigning you back as we will discuss the image loading in today's meeting. 
While you can look into my above suggestion for using solay system data.

Reply:
We can't use same image for multiple explorations,
each exploration uses its id to access the images.

Reply:
ok, in that case, I don't think we can reduce the resources usage. Let's go with the current android image only.